src/packaging/packages.targets

```diff
@@ -38,6 +38,8 @@
             <ILCompilerFiles Include="ILCompiler.MetadataTransform.dll" />
             <ILCompilerFiles Include="ILCompiler.MetadataWriter.dll" />
             <ILCompilerFiles Include="ILCompiler.TypeSystem.dll" />
+            <!-- This is needed for VS Code debugging for ILC - it only likes dll extension -->
```

Comment:
Sure. I was a bit concerned that we might depend on ILC.exe working on desktop (in the desktop project), but upon a quick look it is a separate project after all. I've changed the output to be a EXE with .dll extension (otherwise it won't have the entrypoint)